### ISSUE TITLE
[QUESTION]: train/test/validate split and hyperopt

### ISSUE THREAD

**Issue by kozaczki** [chemprop/chemprop#389]
hi there. Say I would like to run hyperopt script to optimize parameter. three questions here:

As per documentation:

1)  n is the number of hyperparameters settings to try' - does n refer to each of: hidden size, depth, dropout, and number of feed-forward layers parameters or to number of combination of these parameters to try

2) when using the hypertop script and later training model what happens to default train / validation / test split setting? If I understand the documentation properly, the whole dataset will be used for training. 

3) To provide extra features (on molecule and atom level) I need to use the same arguments as for training?

**Comment by cjmcgill** [comment]
1. The number that you provide with `--num_iters` is going to refer to the number of combinations of them to try. You can set which parameters you want to include in your search space with `--search_parameter_keywords` and then listing the keywords you want to search across.
2. The default 80/10/10 splitting will still occur on whatever data you provide. If you want to keep your test set excluded, you will need to do some manual splitting ahead of time.
3. You can provide extra features in the same way that you would during training. In fact, you can specify any of the training arguments when starting a hyperparameter optimization job. Those will all be passed along to the individual trials. So things like your extra features or the number of epochs that you want to train for can be included in your trials.

**Comment by kevingreenman** [comment]
Closing this issue now, feel free to re-open if you have additional questions.
